splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -20,6 +20,8 @@
 import subprocess
 import sys
 from pathlib import Path
+
+from packaging.version import Version
```

Comment:
Can you please take a look at https://github.com/splunk/addonfactory-ucc-generator/pull/973? I think we might need to add this as an explicit dependency to the `pyproject.toml` so we know we depend on it 100%.